plugins/ping.py

```diff
@@ -59,7 +59,7 @@ def ping(text, reply):
         r = int(m.group(2)) - int(m.group(1))
         return "min: %sms, max: %sms, average: %sms, range: %sms, count: %s" \
                % (m.group(1), m.group(2), m.group(3), r, count)
-    else:
-        m = re.search(unix_ping_regex, pingcmd)
-        return "min: %sms, max: %sms, average: %sms, range: %sms, count: %s" \
-               % (m.group(1), m.group(3), m.group(2), m.group(4), count)
+
+    m = re.search(unix_ping_regex, pingcmd)
+    return "min: %sms, max: %sms, average: %sms, range: %sms, count: %s" \
```

Comment:
This is inconsistent with formatting used else where, please fix it here or in a new PR

Reply:
I'm limiting the changes to what I set out to do, specifically just removing the `return-else` statements

Reply:
This also falls under "own PR"